src/libraries/System.Net.NetworkInformation/tests/FunctionalTests/NetworkInterfaceBasicTest.cs

```diff
@@ -294,8 +294,8 @@ public async Task NetworkInterface_LoopbackInterfaceIndex_MatchesReceivedPackets
                 }
 
                 Assert.Equal(
-                    (await receivedTask).PacketInformation.Interface,
-                    ipv6 ? NetworkInterface.IPv6LoopbackInterfaceIndex : NetworkInterface.LoopbackInterfaceIndex);
+                    ipv6 ? NetworkInterface.IPv6LoopbackInterfaceIndex : NetworkInterface.LoopbackInterfaceIndex,
+                    (await receivedTask).PacketInformation.Interface);
```

Comment:
Switched order of these as expected should be first so the error message was confusing me